app/shared/resource-type-filter/ResourceTypeFilterContainer.js

```diff
@@ -0,0 +1,49 @@
+import includes from 'lodash/includes';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import ResourceTypeFilterButton from './ResourceTypeFilterButton';
+
+export class UnconnectedResourceTypeFilterContainer extends Component {
+  static propTypes = {
+    onFilterResourceType: PropTypes.func.isRequired,
+    onUnfilterResourceType: PropTypes.func.isRequired,
+    resourceTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+    filteredResourceTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(resourceType) {
+    const {
+      filteredResourceTypes,
+      onFilterResourceType,
+      onUnfilterResourceType,
+    } = this.props;
+    if (includes(filteredResourceTypes, resourceType)) {
+      onUnfilterResourceType(resourceType);
+    } else {
+      onFilterResourceType(resourceType);
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        { this.props.resourceTypes.map(resourceType =>
+          <ResourceTypeFilterButton
+            active={!includes(this.props.filteredResourceTypes, resourceType)}
+            key={`resource-type-${resourceType}`}
+            onClick={this.handleClick}
+            resourceType={resourceType}
+          />
+        )}
+      </div>
+    );
+  }
+}
+
+export default connect()(UnconnectedResourceTypeFilterContainer);
```

Comment:
The selectors and actions were here at the beginning, but I moved them to the AdminResourcesPage so we could reuse this component if needed. So I'll just return the component without "connect"